src/core/Form/FileInput/FileInput.baseStyles.tsx

```diff
@@ -143,6 +143,10 @@ export const baseStyles = (
           display: flex;
           align-items: center;
 
+          &.fi-file-input_input_wrapper--hidden {
```

Comment:
The classname in CSS and in FileInput.tsx do not match. `input_wrapper--hidden` (here) vs `input-wrapper--hidden` (in FileInput.tsx). The former is used consistently in CSS but the latter form (with a hyphen between input and wrapper) would match the class used for the base state of the wrapper.

Reply:
Good point, fixed now